tox.ini

```diff
@@ -97,6 +97,10 @@ commands =
     {cuda,cuda11x,cuda12x}: python -m pytest --cache-clear -v -n {env:NUM_PROCESSES:1} -m "requires_gpu" {posargs} tests{/}storage_tests
     #pytest doctest-modules {posargs} src{/}gt4py{/}storage
 
+[testenv:notebooks-py{310,311}]
```

Comment:
I think `[testenv:requirements-{common,py38,py39,py310}]` should be updated to `[testenv:requirements-{common,py38,py39,py310,py311}]` or am I missing something? 
(Not really related to this PR, but since `tox.ini` is modified anyway, it could be a good opportunity to fix it).

Reply:
done